docs/user-guide/story-setting.md

```diff
@@ -1,29 +1,43 @@
-
- 
-The Story Settings panel allows the editor to configure which additional components should appear to end users in the story view. The **Title**, a **Logo** and the **Navigation Toolbar**, if properly configured, will appear in the top bar of the story itself while the colors of the different components in the story can be customized in **Default Theme** and **Overlay** section of the story settings.
+The Story Settings panel allows the editor to customize the theme of the story and configure which additional components should appear to end users in the story view.
 
 The Story settings panel is available in Edit Mode and it can be opened by clicking on the **Settings** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.
 
-
-
 <img src="../img/story-setting/story-setting-1.jpg" class="ms-docimage" style="max-width:400px"/>
 
-The editor can enable or disable and edit the following components:
+## Story Theme
 
-* The **Title** of the story, the default value is the title given to the story's resource in MapStore
+The editor can customize the different components of the story through the following sections:
 
-* The **Theme** to choose the default background and text color of the whole story: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color: 
+* The **Theme** to choose the default background and text color and the default font of the whole story: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color: 
 
 <img src="../img/story-setting/theme.gif" class="ms-docimage"/>
 
-* The **Overlay** to choose the default background and text color of the *Contents* present in the [Title Section](title-section.md#content) and in the [Immersive Section](immersive-section.md#content) as well: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/> a color picker appears to allow selecting the desired color:
+ * The *Font Family* to choose the default text font present in the whole story: clicking on the search bar a dropdown menu opens to allow selecting the desired font (*Inherit*, *Arial*, *Georgia*, *Impact*, *Tahoma*, *Time New Roman*, *Verdana*): 
```

Comment:
We should put a note here (with a link to the related configration in dev guide) to notify that the list of available fonts families can be customized by configuration file.